app/views/welcome/_faq.html.erb

```diff
@@ -2,15 +2,15 @@
   <div class="container">
 
     <div class="section-header">
-      <h2>F.A.Q </h2>
+      <h2>FAQ</h2>
     </div>
 
-    <div class="row justify-content-center">
-      <div class="col-lg-9">
+    <div class="row">
+      <div class="col">
           <ul id="faq-list">
 
             <li>
-              <a data-toggle="collapse" class="collapsed" href="#faq1">Non consectetur a erat nam at lectus urna duis? <i class="fa fa-minus-circle"></i></a>
+              <a data-toggle="collapse" href="#faq1" class="collapsed">Non consectetur a erat nam at lectus urna duis? <i class="fa fa-minus-circle"></i></a>
               <div id="faq1" class="collapse" data-parent="#faq-list">
                 <p>
                   Feugiat pretium nibh ipsum consequat. Tempus iaculis urna id volutpat lacus laoreet non curabitur gravida. Venenatis lectus magna fringilla urna porttitor rhoncus dolor purus non.
```

Comment:
I think that the font size for F.A.Q. (in mobile version) is a bit too small as of now? Probably can consider to make it a bit bigger